setup.sh

```diff
@@ -19,25 +19,28 @@ function getLatestK8s {
    echo $latestVersion
 }
 
-# Environment variables / positional parameters and defaults. 
+# Environment variables / positional parameters and defaults.
+
+# Add ~/bin to PATH for the script only 
+export PATH=$PATH:~/bin
 
 # TODO make $1 and $2 required?
-keyName=$1
+read -p "Enter the key name used to sign and verify: "  keyName
 
-keySubjectName=$2
+read -p "Enter the key subject name: "  keySubjectName
 
-rgName=$3
+read -p "Enter the resource group to deploy into: "  rgName
 rgName=${rgName:-myakv-akv-rg}
 
 # The location to store the meta data for the deployment.
 location=${location:-southcentralus} # Currently only region to support premium ACR with Zone Redundancy
 
 # The version of k8s control plane
-k8sversion=$4
+read -p "Enter the Kubernetes version to use: "  k8sversion
 k8sversion=${k8sversion:-$(getLatestK8s)}
 
 # Environment Name (ACR, AKS, KV prefix)
-envPrefix=$5
+read -p "Enter the environment name: "  envPrefix
```

Comment:
same as comment on line 32 and 39

Maybe we can talk through it in our team meeting and see other perspectives.